examples/strings/demo/cpp/caesar.cpp

```diff
@@ -0,0 +1,15 @@
+typedef long int i32;
```

Comment:
Can we rename the parent folder from `/cpp/` to `/c/` I forgot about that :upside_down_face: Thanks! :+1:

Reply:
Done!